gpbft/participant.go

```diff
@@ -161,10 +161,14 @@ func (p *Participant) beginInstance() error {
 	if err != nil {
 		return fmt.Errorf("failed fetching power table for instance %d: %w", p.nextInstance, err)
 	}
+	data, err := p.host.GetDataForInstance(p.nextInstance)
```

Comment:
I think this could be merged into GetChainForInstance (could be renamed as GetProposalForInstance). It's needed at exactly the same time and for essentially the same reason.

Reply:
Fair enough. That resolves https://github.com/filecoin-project/go-f3/pull/273#discussion_r1614016791.

Reply:
@Kubuxu SGTY? Also, I think the line between `GetComitteeForInstance` and `GetChainForInstance` is getting a bit blurry.

Reply:
> Also, I think the line between `GetComitteeForInstance` and `GetProposalForInstance` is getting a bit blurry.

Nvm. I now understand this a bit better (we `GetComitteeForInstance` is what we absolutely _need_ for validation).